src/transformers/utils/dummy_vision_objects.py

```diff
@@ -310,6 +310,20 @@ def __init__(self, *args, **kwargs):
         requires_backends(self, ["vision"])
 
 
+class Kosmos2_5ImageProcessor(metaclass=DummyObject):
+    _backends = ["vision"]
+
+    def __init__(self, *args, **kwargs):
+        requires_backends(self, ["vision"])
+
+
+class Kosmos2_5Processor(metaclass=DummyObject):
+    _backends = ["vision"]
+
+    def __init__(self, *args, **kwargs):
+        requires_backends(self, ["vision"])
```

Comment:
Ah, I see. I am discussing something internally with the team members. Let's keep `Kosmos2_5Processor` here and discard what I mentioned above in `__init__` file for now.